nipoppy_cli/nipoppy/workflows/dataset_init.py

```diff
@@ -94,11 +92,18 @@ def run_main(self):
             " to match your dataset"
         )
 
+    def run_cleanup(self):
+        """Log a success message."""
+        self.logger.info(
+            f"[green]Successfully initialized a dataset at {self.dpath_root}![/]"
```

Comment:
Similarly the colors for the different levels of success, partial success, failures: should they be extracted into a global variable of some sort?

This may be over engineering on my part...

Reply:
Added return codes and colours in nipoppy.env module